src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicator.java

```diff
@@ -183,7 +204,175 @@ public void initAverRestCommunicator() throws Exception {
 	 */
 	@Override
 	public void controlProperty(ControllableProperty controllableProperty) {
+		String property = controllableProperty.getProperty();
+		String value = String.valueOf(controllableProperty.getValue());
+		controlOperationsLock.lock();
+
+		try {
+			// Power
+			if (property.startsWith(Command.POWER.getName())) {
```

Comment:
As we discussed locally, you should parse this into enum and use the switch case.

Reply:
ok thank you